transformer/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties

```diff
@@ -48,41 +92,15 @@ javax.servlet.jsp.jstl.tlv=jakarta.servlet.jsp.jstl.tlv
 javax.servlet.jsp.tagext.JspFragment=jakarta.servlet.jsp.tagext.JspFragment
 javax.servlet.jsp.jstl=jakarta.servlet.jsp.jstl
 
-javax.json=jakarta.json
-javax.json.spi=jakarta.json.spi
-javax.json.stream=jakarta.json.stream
-
-javax.json.bind=jakarta.json.bind
-javax.json.bind.adapter=jakarta.json.bind.adapter
-javax.json.bind.annotation=jakarta.json.bind.annotation
-javax.json.bind.serializer=jakarta.json.bind.serializer
-javax.json.bind.spi=jakarta.json.bind.spi
-javax.json.bind.config=jakarta.json.bind.config
- 
-javax.enterprise.context.spi=jakarta.enterprise.context.spi
-javax.enterprise.inject.spi=jakarta.enterprise.inject.spi
-javax.decorator=jakarta.decorator
-javax.enterprise.inject=jakarta.enterprise.inject
-javax.enterprise.context=jakarta.enterprise.context
-javax.enterprise.context.control=jakarta.enterprise.context.control
-javax.enterprise.event=jakarta.enterprise.event
-javax.enterprise.inject=jakarta.enterprise.inject
-javax.enterprise.inject.literal=jakarta.enterprise.inject.literal
-javax.enterprise.inject.se=jakarta.enterprise.inject.se
-javax.enterprise.inject.spi.configurator=jakarta.enterprise.inject.spi.configurator
-javax.enterprise.util=jakarta.enterprise.util
-
-javax.interceptor=jakarta.interceptor
-
-javax.inject=jakarta.inject
+javax.transaction=jakarta.transaction
```

Comment:
Should this separate the (Java standard edition) javax.transaction.xa classes?

Reply:
Hi; by "separate" do you mean have an entry:
`javax.transaction.xa=jakarta.transaction.xa
My understanding is this is a special case: `javax.transaction` is renamed, whereas `javax.transaction.xa` is not transformed.
That's a reason why package name matching doesn't match subpackages; otherwise, the rename of `javax.transaction` would catch `xa`.

Reply:
Oh, thanks, I thought that the javax.transaction in this file, might of also matched the javax.transaction.xa subpackage, thanks for explaining! :)